### PR TITLE
Add basic queue system and a class for representing users.

### PR DESCRIPTION
**Pull request by simon-andrews**
This pull request adds a basic queue system. There are still some kinks that may need to be ironed out, but I tested it and it is functional.

### CODE REVIEW

**Comment by Math-ias** [comment]
What will Server.java be for?

**Comment by simon-andrews** [comment]
Represents the Run-Queue-Server itself. This will probably contain the `main(String[])` method.

**Comment by TheLocust3** [comment]
Just a heads up, I am going through and reviewing all PRs. Next time please assign me when you are done with a PR so I will review it.

File: src/org/robockets/runqueue/server/User.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	private String name;
+	private Calendar timeLastJoined;
+	/**
+	 * Constructor for creating new users.
+	 * @param name Username for the user
+	 */
+	public User(String name) {
+		this.name = name;
+	}
+	/**
```
Comment (TheLocust3) [inline src/org/robockets/runqueue/server/User.java]: Put a newline in-between the `}` and the start of a comment `/**`

**Comment by simon-andrews** [comment]
If someone else was interested in reviewing it (because they were going to work with/on that code and wanted a say on how it worked) then I left the option open for them to self-assign.

**Comment by TheLocust3** [comment]
Then next time please announce it. Anyways there should be a space in between a function name and your arguments. Check your other PR for an example.

**Comment by TheLocust3** [comment]
+1

**Pull request merged**